tests/src/JIT/Regression/JitBlue/GitHub_5047/project.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
It wouldn't hurt. The reason that we have common project.json files is to speed up package restore--common app.configs don't have any similar compile or run time benefit that I'm aware of, so we did not do that work earlier.